<?xml version="1.0"?>
<launch>
    <node name="semantic_node" pkg="social_segway" type="semantic_node" output="screen"/>
        <param name="compare" value="$(arg compare)" />
    </node>
    <group unless="$(arg compare)">
        <node name="main_node" pkg="social_segway" type="main_node" output="screen"/>
    </group>
</launch>
